data/output/result

```diff
@@ -0,0 +1,29 @@
+&~2Ch{E*nf  00000000000000000000000000000011  6666DDDDFFFFFFFF4444EEEE11116666BBBB77776666BBBB6666
```

Comment:
include data file?

Reply:
Just in case, your current files are different from this. So this can be removed any time if we merge code right.